data_collection/web/guides.rst

```diff
@@ -9,4 +9,224 @@ Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem
 Downloads and Outlinks
 ----------------------
 
-Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop p
+.. note::
+  If you want to use CSS classes or HTML attributes to mark links as download or outlink and you have modified default JS snippet, make sure that :ref:`enableLinkTracking<tracking-outlink>` is called. It is enabled in default snippet, but if you use a custom one, then you have to enable it by yourself.
+
+```
+// Enable Download & Outlink tracking
```

Comment:
Without any text under it I would think I need to copy it to my tracking code - but I do not need to. 

Downloads and Outlinks should work by default in 95% of cases. The rest are customs. Please add information about it.

Reply:
There is such info in a text above `It is enabled in default snippet`

Reply:
It is now lower in the document, so it is ok